CMakeLists.txt

```diff
@@ -4,15 +4,15 @@ project("gpu-filters")
 # -- Config -- #
 set(DEBUG_FLAGS -"D_GLIBCXX_DEBUG")
 set(CMAKE_C_FLAGS "-std=c99")
-set(CMAKE_CXX_FLAGS "-std=c++11 -Wconversion -Wshadow -Wall -Wextra -Wno-missing-braces -DCL_HPP_TARGET_OPENCL_VERSION=200 -DCL_HPP_MINIMUM_OPENCL_VERSION=200")
+set(CMAKE_CXX_FLAGS "-std=c++11 -Wconversion -Wshadow -Wall -Wextra -Wno-missing-braces -DCL_HPP_TARGET_OPENCL_VERSION=120 -DCL_HPP_MINIMUM_OPENCL_VERSION=120")
```

Comment:
Could you add a comment to make sure we don't change the version without checking whether it runs on mac?